kolibri/plugins/coach/assets/src/state/actions/reports.js

```diff
@@ -34,11 +33,11 @@ const SummaryLogResource = coreApp.resources.ContentSummaryLogResource;
  * @returns {Promise} that resolves channel with lastActive value in object:
  *   { 'channelId': dateOfLastActivity }
 */
-function channelLastActivePromise(channel, classId) {
+function channelLastActivePromise(channel, userScope, userScopeId) {
   const summaryPayload = {
     channel_id: channel.id,
-    collection_kind: ReportConstants.UserScopes.CLASSROOM,
-    collection_id: classId,
+    collection_kind: userScope,
```

Comment:
`userIdOrUserCollectionId`? :)

for expediency I'm going to leave this for now if that's cool - but I'm happy to revisit